Patch-Source: https://github.com/CrossNox/m2r2/pull/22
From 37832c0254a2413ee666a42b6906361289b50202 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Christian=20H=C3=A4ggstr=C3=B6m?=
 <christian.haggstrom@orexplore.com>
Date: Wed, 27 Jan 2021 14:23:25 +0100
Subject: [PATCH] Don't try to import m2r2 during setup

Don't try to import m2r2 during setup, since it expects m2r2 to be
installed already.

Fixes the following error during pip install:

pkg_resources.DistributionNotFound: The 'm2r2' distribution was not found and is required by the application
---
 setup.py | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/setup.py b/setup.py
index 4c6c913..23de72a 100644
--- a/setup.py
+++ b/setup.py
@@ -10,13 +10,8 @@
     from distutils.core import setup
 
 readme_file = path.join(path.dirname(path.abspath(__file__)), "README.md")
-try:
-    from m2r2 import parse_from_file
-
-    readme = parse_from_file(readme_file)
-except ImportError:
-    with open(readme_file) as f:
-        readme = f.read()
+with open(readme_file) as f:
+    readme = f.read()
 
 
 __version__ = "0.2.5"
